app/controllers/ems_common.rb

```diff
@@ -571,12 +571,14 @@ def recheck_authentication(id = nil)
   end
 
   def check_compliance(model)
-    emss = find_checked_items
-    if emss.empty?
+    showlist = @lastaction == "show_list"
+    ids = showlist ? find_checked_items : ([params[:id]] if model.find(params[:id]))
```

Comment:
it doesn't seem to fit, we want to have the ids in a list, but this will return a single value if the list length is 1.